ArduCopter/version.h

```diff
@@ -1,12 +1,12 @@
 #pragma once
 
-#include "ap_version.h"
```

Comment:
My bad.  Apparently that got lost between brain and keyboard.  I've put it in there now.